java/java11/Dockerfile

```diff
@@ -19,6 +19,14 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends libxml2-utils && \
     rm -rf /var/cache/apt
 
+# Add Graphviz
+RUN set -ex; \
```

Comment:
We should be able to accomplish this behavior using `&&` between the commands to match the style of the other docker build steps